app/templates/settings/general.hbs

```diff
@@ -71,7 +71,7 @@
             </div>
             <div class="gh-setting-action gh-setting-action-smallimg">
                 {{#if model.icon}}
-                    <img class="blog-icon" src="{{model.icon}}" alt="icon" data-test-icon-img>
+                    <img class="blog-icon" src="{{model.icon}}" onClick={{action "triggerFileDialog"}} alt="icon" data-test-icon-img>
```

Comment:
@kevinansfield Shall we refactor the `onClick` in the `<button>` tags then as well?

Like here: https://github.com/TryGhost/Ghost-Admin/blob/master/app/templates/settings/general.hbs#L81